src/interfaces/store-interfaces/state-interfaces/flags-company-info-interface.ts

```diff
@@ -21,4 +21,5 @@ export interface FlagsCompanyInfoIF {
   isValidCreateSpecialResolution: boolean
   isValidApprovalType: boolean
   isValidExtensionTime: boolean
+  isValidRelationship?: boolean
```

Comment:
The other ones are required and True by default. Why is this one different?

Reply:
Ah, well spotted, I'll fix it.